src/MCPServer/lib/linkTaskManagerDirectories.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, jobChainLink, pk, unit):
         # Apply unit (SIP/Transfer) replacement values
         commandReplacementDic = unit.getReplacementDic(directory)
         # Escape all values for shell
-        for key, value in commandReplacementDic.items():
+        for key, value in list(commandReplacementDic.items()):
```

Comment:
The fixer automatically converts from `d.items()` to `list(d.items())`, since in py2 it's explicitly a list and the fixer is trying to preserve that behaviour. I reverted the ones I caught that didn't need that fix, thanks for catching the ones I missed.